### PR TITLE
chore: 🤖 bump ui-core to fix accordion animation glitch

### PR DESCRIPTION
**Pull request by apryka**
https://metakeen.atlassian.net/browse/FEE-734

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1736,10 +1729,10 @@
   dependencies:
     "@keen.io/colors" "^1.1.0"
 
-"@keen.io/parser@^2.8.77":
-  version "2.8.77"
-  resolved "https://registry.yarnpkg.com/@keen.io/parser/-/parser-2.8.77.tgz#10615da365cc81957d160c5d96923e9ff8bba285"
-  integrity sha512-xn9b7FMcDuK2fpsMGOpuszuSI+sToOGtFMCiobqk483vmi4OfO2zafdIDdmeujcsN5reCfioD3OJyELOuii30A==
+"@keen.io/parser@^2.8.77", "@keen.io/parser@^2.8.79":
```
Comment (maciejrybaniec) [inline yarn.lock:1732]: Could we check why the two versions of @keen.io/parser are required ?

**Review by maciejrybaniec — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (apryka) [reply yarn.lock:1732]:
```yarn why v1.19.0
[1/4] 🤔  Why do we have the module "@keen.io/parser"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "@keen.io/parser@2.8.79"
info Has been hoisted to "@keen.io/parser"
info Reasons this module exists
   - Specified in "devDependencies"
   - Hoisted from "@keen.io#dataviz#@keen.io#parser"
info Disk size without dependencies: "556KB"
info Disk size with unique dependencies: "1.08MB"
info Disk size with transitive dependencies: "3.59MB"
info Number of shared dependencies: 4
✨  Done in 0.82s.
(Keen-Web) Arturs-MacBook-Pro:dashboard-creator arturpryka$ yarn dedupe
yarn dedupe v1.19.0
error The dedupe command isn't necessary. `yarn install` will already dedupe.

Comment (apryka) [reply yarn.lock:1732]: `yarn dedupe` after migration should fix the issue

Comment (maciejrybaniec) [reply yarn.lock:1732]: Ok 👍

**Review by maciejrybaniec — APPROVED** [review]

**Pull request merged**